src/FacadeAccountCreation/FacadeAccountCreation.Core/Services/Organisation/OrganisationService.cs

```diff
@@ -279,4 +279,31 @@ public async Task UpdateOrganisationDetails(
 
         response.EnsureSuccessStatusCode();
     }
+
+    public async Task<List<OrganisationNationModel>> GetOrganisationNationByExternalIdAsync(Guid organisationExternalId)
```

Comment:
Please consider changing your return type to be nullable complient if any of your code path returns null, 
another way to put it, should your code return nulls?

Reply:
done